pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -0,0 +1,629 @@
+// api.pb.go could be generate by hack/update-generate-runtime.sh
+syntax = 'proto2';
```

Comment:
@yifan-gu Yes, there are no `optional` keyword in proto3. If we want to switch proto2 to proto3, we should also remove all `optional` keyword in `api.proto`.